runtime/src/launch/slurm-srun/launch-slurm-srun.c

```diff
@@ -421,8 +436,10 @@ static char* chpl_launch_create_command(int argc, char* argv[],
 
     // set the job name
     len += snprintf(iCom+len, sizeof(iCom)-len, "--job-name=%s ", jobName);
-    // suppress informational messages, will still display errors
-    len += snprintf(iCom+len, sizeof(iCom)-len, "--quiet ");
+    if(!getSlurmDebug(false)) {
```

Comment:
Formatting nit, add space between `if(`

Sometimes in the runtime code we'll also do fake named args like `getSlurmDebug(/*debug=*/false)` to make it more obvious what true/false args represent. No strong preference for that, so your call if you find that style helpful.

Reply:
Fixed.